index.js

```diff
@@ -0,0 +1,58 @@
+function calculate(calculationSteps) {
+  if (calculationSteps.length < 3) return 0
+  let operation = ['/', '+', '-', '*']
+  let num1, num2 = ''
+
+  // operation = calculationSteps.find((calculationStep) => (isNaN(calculationStep) && (operation.indexOf(calculationStep) >= 0)))
+
+  calculationSteps = calculationSteps.filter(calculationStep => (!isNaN(calculationStep) && calculationStep !== '') || (isNaN(calculationStep) && (operation.indexOf(calculationStep) >= 0)))
+
+  num1 = (calculationSteps[0] === null) ? 0 : parseInt(calculationSteps[0])
+  operation = calculationSteps[1]
+  num2 = (calculationSteps[2] === null) ? 0 : parseInt(calculationSteps[2])
+
+  return findAns(num1, num2, operation)
+}
+
+function findAns(num1, num2, operation) {
+  switch (operation) {
+    case '+': return parseInt(num1) + parseInt(num2)
+    case '-': return parseInt(num1) - parseInt(num2)
+    case '*': return parseInt(num1) * parseInt(num2)
+    case '/': return (isFinite(parseInt(num1.value) / parseInt(num2.value))) ? parseInt(num1) / parseInt(num2) : 0
+    default: return NaN
+  }
+}
+
+module.exports = calculate
+
+function calculate(calculationSteps) {
+  if (calculationSteps.length < 3) return 0
+  let operation = ['/', '+', '-', '*']
+  let num1, num2 = ''function calculate(calculationSteps) {
+    if (calculationSteps.length < 3) return 0
+    let operation = ['/', '+', '-', '*']
+    let num1, num2 = ''
+
+    // operation = calculationSteps.find((calculationStep) => (isNaN(calculationStep) && (operation.indexOf(calculationStep) >= 0)))
+
+    calculationSteps = calculationSteps.filter(calculationStep => (!isNaN(calculationStep) && calculationStep !== '') || (isNaN(calculationStep) && (operation.indexOf(calculationStep) >= 0)))
+
+    num1 = (calculationSteps[0] === null) ? 0 : parseInt(calculationSteps[0])
+    operation = calculationSteps[1]
+    num2 = (calculationSteps[2] === null) ? 0 : parseInt(calculationSteps[2])
+
+    return findAns(num1, num2, operation)
+  }
+
+  function findAns(num1, num2, operation) {
+    switch (operation) {
+      case '+': return parseInt(num1) + parseInt(num2)
+      case '-': return parseInt(num1) - parseInt(num2)
+      case '*': return parseInt(num1) * parseInt(num2)
+      case '/': return (isFinite(parseInt(num1.value) / parseInt(num2.value))) ? parseInt(num1) / parseInt(num2) : 0
+      default: return NaN
+    }
+  }
+
+  module.exports = calculate
```

Comment:
It seems you've accidentally pasted a duplicate of your code here by mistake before pushing to github, but upon deletion of lines 28-58 everything works fine, so no worries, its not something I would dock points for, just be sure to double-check for things like this in the future.